contracts-periphery/src/test/BatchSendwToll.t.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./UmbraBatchSend.t.sol";
+
+contract BatchSendwTollTest is UmbraBatchSendTest {
+
+    address mockOwner;
+
+    function setUp() public override {
+
+        vm.label(alice, "Alice");
+        vm.label(bob, "Bob");
+        vm.label(address(this), "UmbraBatchSendTest");
```

Comment:
Did you find that the test contract was not labeled automatically in traces? It should be, and if so we can remove this